app/src/main/res/layout/view_background_message.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <TextView
+        android:id="@+id/messageTextView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="16dp"
+        android:drawablePadding="8dp"
```

Comment:
I think the text can have a little more spacing to the drawable and the button, maybe 12-16dp? Also please add a left/right padding and a `android:lineSpacingMultiplier="1.1"` because the text is going to get longer